MedtrumKit/Encryption/WritePacket.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  WritePacket.swift
+//  MedtrumKit
+//
+//  Created by Bastiaan Verhaar on 26/02/2025.
+//
+
+class WritePacket {
```

Comment:
Add unit tests for this thing, It can have nasty surprises ;)

Reply:
Done!